config/samples/datadoghq_v2alpha1_datadogpodautoscaler.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: datadoghq.com/v2alpha1
```

Comment:
This can also be removed as we don't ship the CRD with the operator

Reply:
Done